src/download.jl

```diff
@@ -0,0 +1,51 @@
+"""
+dataurl(dataset_code)
+This function takes in a dataset code of format "GADM/<country_code>"
+Returns the URL of the gpkg dataset of the country
+"""
+function dataurl(dataset_code)
```

Comment:
I would call this function `gadmurl` or a similar name given that it is for a specific dataset with a specific URL convention.

Reply:
Since we plan to add more URLs later on, I kept it for generality.